spec/integration/_dev/benchmarks/scenario.spec.yml

```diff
@@ -0,0 +1,24 @@
+spec:
+  type: object
+  additionalProperties: false
+  required:
+    - version
+    - data_stream
+    - description
+    - corpora
+  properties:
+    version:
+      description: The package version to benchmark.
+    data_stream:
+      description: The data-stream to benchmark.
+    description:
+      description: A description of the benchmark scenario.
+    corpora:
+      description: Corpora settings.
+      type: object
+      additionalProperties: false
+      properties:
+        input_service:
+          description: The service (found in ./deploy/docker) that will send corpora logs to the elastic agents.
+    warmup_time_period:
```

Comment:
So, would it be like adding as default value 0?

```yaml
warmup_time_period:
  type: integer
  default: 0
```
Depending on the type for this field (according to test packages) it could be:
```yaml
warmup_time_period:
  type: string
  default: 0s
```

It could also be added this behavior into the description of the field too.